samples/Friends/Friends/Friends.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <TslCodegen Include="**/*.tsl" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphEngine.Core" Version="2.0.0" />
+    <PackageReference Include="GraphEngine.Core" Version="2.0.10084" />
```

Comment:
please revert. 10084 is a specific build and may not exist.

Reply:
Done